src/context/App.tsx

```diff
@@ -43,6 +43,8 @@ export const defaultSettings: SettingsState = {
   detailedNotifications: false,
   markAsDoneOnOpen: false,
   showAccountHostname: false,
+  kbdShortcut: 'CmdOrCtrl+Alt+G',
```

Comment:
Constants may be a better home for this :)

Reply:
Yup good idea, I'll create a constant that will look something like `DEFAULT_KBD_SHORTCUT` 👍